core/modules/path/path.module

```diff
@@ -714,7 +714,11 @@ function path_form_element_validate($element, &$form_state, $complete_form) {
     }
     $query->addExpression('1');
     $query->range(0, 1);
-    if ($query->execute()->fetchField()) {
+
+    // Ensure that the submitted alias is not a system path.
+    $is_system_path = !empty(menu_get_item($path['alias']));
```

Comment:
empty() can only be done on a variable, so this would need to be adjusted to:
```
$menu_item = menu_get_item($path['alias']);
$is_system_path = !empty($menu_item);
```